packages/sdk-react/src/mocks/counterparts/address/counterpartsAddressesFixture.ts

```diff
@@ -1,18 +1,16 @@
+import { components } from '@/api';
 import { counterpartListFixture } from '@/mocks';
 import { getRandomNumber, getRandomProperty } from '@/utils/storybook-utils';
 import { faker } from '@faker-js/faker';
-import {
-  CounterpartAddressResourceList,
-  AllowedCountries,
-  CounterpartAddress,
-} from '@monite/sdk-api';
+import { AllowedCountries } from '@monite/sdk-api';
 
 import { organizationId } from '../counterpart.mocks.types';
 
 function generateRandomAddress(
   counterpartId: string
-): CounterpartAddressResourceList {
+): components['schemas']['CounterpartAddressResourceList'] {
   return {
+    // @ts-expect-error - check how to get property for country without enums
```

Comment:
Could you please clarify the problem? If it can be solved within this PR - that would be cool.